### DISCUSSION TITLE
Removing old version

### DISCUSSION THREAD

**Discussion by Tomxtal** [hmatuschek/qdmr#480] [General]
I have source build 0.11.2 on Debian 12.7. I would like update to version 0.12. Do I need to remove the old version before building the new version. If so is there a easy to remove the old version?

**Comment by hmatuschek** [comment]
The majority of files will be overridden by the new install (given, you use the same prefix). However, there will be some libraries left. So, maybe delete `libdmrconf*`  first.

**Reply by Tomxtal** [reply]
Hannes thank for your help. I have now compiled 0.12 and working ok will be doing some testing over the next few weeks.
